docs/code.rst

```diff
@@ -216,6 +216,9 @@ Operators
 .. autoclass:: airflow.contrib.operators.gcp_vision_operator.CloudVisionProductSetGetOperator
 .. autoclass:: airflow.contrib.operators.gcp_vision_operator.CloudVisionProductSetUpdateOperator
 .. autoclass:: airflow.contrib.operators.gcp_vision_operator.CloudVisionProductUpdateOperator
+.. autoclass:: airflow.contrib.operators.gcp_video_intelligence_operator.CloudVideoIntelligenceDetectVideoLabelsOperator
```

Comment:
Please keep alphabetical order. You should sort first by classname. In next step, by full class path.

Reply:
Done

Reply:
Done